sys/net/routing/aodvv2/aodv.c

```diff
@@ -477,6 +463,32 @@ static void _write_packet(struct rfc5444_writer *wr __attribute__ ((unused)),
     AODV_DEBUG("%d bytes sent.\n", bytes_sent);
 }
 
+/* Print the json representation of a sent packet to stdout for debugging */
+static void print_json_pkt_sent(struct writer_target *wt)
+{
+    // note: what if the content at wt has changed until this is printed? memcpy the entire thing?
+    int msg_type = wt->type;
+    if (msg_type == RFC5444_MSGTYPE_RREQ) {
+    LOG("{\"log_type\": \"sent_rreq\", \"log_data\": {"
+        "\"orig_addr\": \"%s\", \"targ_addr\": \"%s\", \"orig_seqnum\": %d, \"metric\": %d}}\n",
+        netaddr_to_string(&nbuf_oa, &wt->packet_data.origNode.addr),
+        netaddr_to_string(&nbuf_ta, &wt->packet_data.targNode.addr),
+        wt->packet_data.origNode.seqnum, wt->packet_data.origNode.metric);
+    }
+    if (msg_type == RFC5444_MSGTYPE_RREP) {
+    LOG("{\"log_type\": \"sent_rrep\", \"log_data\": {"
+        "\"next_hop\": \"%s\",\"orig_addr\": \"%s\", \"orig_seqnum\": %d,"
+        " \"targ_addr\": \"%s\"}}\n",
+                netaddr_to_string(&nbuf_nh, &wt->next_hop),
+                netaddr_to_string(&nbuf_oa, &wt->packet_data.origNode.addr),
+                wt->packet_data.origNode.seqnum,
+                netaddr_to_string(&nbuf_ta, &wt->packet_data.targNode.addr));
+    }
+    if (msg_type == RFC5444_MSGTYPE_RERR) {
+        /* TODO */
```

Comment:
Perhaps log something here anyway so that once notices that this is not implemented yet?